ngraph/core/reference/include/ngraph/runtime/reference/prelu.hpp

```diff
@@ -42,6 +43,19 @@ namespace ngraph
                     out[i] =
                         arg[i] < T(0) ? T(arg[i] * slope[cnt++ % shape_size(slope_shape)]) : arg[i];
                 }
+                // assert(arg_shape.size() > 2);
```

Comment:
Why did you add a commented code?

Reply:
@ilyachur  I removed the below commented code. 
`assert(arg_shape.size() > 2);
size_t batch_size = arg_shape[0] * arg_shape[1];
size_t sptial_size = shape_size(arg_shape) / batch_size;
for (int n = 0; n < batch_size; ++n)
{
    T slope_n = shape_size(slope_shape) == 1 ? slope[0] : slope[n % shape_size(slope_shape)];
    for (int sp = 0; sp < sptial_size; ++sp)
    {
        size_t inx = n * sptial_size + sp;
        out[inx] =
            arg[inx] < T(0) ? T(arg[inx] * slope_n) : arg[inx];
    }
}`

This commented code is just for temporal test and communication. When I test prelu as fused post ops in cpu plugin, with slope of size C(channel size), each channel share the same slope. e.g. data[2,8,5,5], slope[1,8,1,1]. Commented code assuming data is planar layout in ngraph and have same result as in cpu plugin. So do we need some broadcast here for prelu or broadcast was done already in other place? please correct me if I am misunderstanding something.

Reply:
@ilyachur I added shape broadcast feature for prelu according [ops specification](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/activation/PReLU_1.md), could you please take a look for ngraph part? Thank you.